include/swift-c/DependencyScan/DependencyScan.h

```diff
@@ -368,6 +368,57 @@ swiftscan_scanner_cache_load(swiftscan_scanner_t scanner,
 SWIFTSCAN_PUBLIC void
 swiftscan_scanner_cache_reset(swiftscan_scanner_t scanner);
 
+//=== Static Mirror Scan Operations ---------------------------------------===//
+
+/// Container of the configuration state for binary static mirror scanning
+/// instance
+typedef void *swiftscan_static_mirror_t;
```

Comment:
Do they have to be in the same header file? I think dependency scanning support and static mirror APIs can evolve independently.
Otherwise, LGTM!

Reply:
Makes perfect sense. The implementation headers were already split.

I refactored the existing API to keep living in `DependencyScan.h`, the new API to live in `BinaryScan.h` and a new umbrella: `LibSwiftScan.h` which ties them together.

Reply:
Thanks!